explainability-core/src/test/java/org/kie/trustyai/explainability/local/shap/ShapKernelExplainerTest.java

```diff
@@ -853,4 +855,51 @@ void testCategoricalCollisions() throws ExecutionException, InterruptedException
         assertEquals(1, shapResults.getSaliencies()[1].getPerFeatureImportance().get(1).getScore(), 1e-6);
     }
 
+    @Test
+    void testToString() throws ExecutionException, InterruptedException, TimeoutException {
+        Random random = new Random(0L);
+
+        List<PredictionInput> background = new ArrayList<>();
+        for (int bg = 0; bg < 1; bg++) {
+            List<Feature> features = new ArrayList<>();
+            for (int i = 0; i < 5; i++) {
+                if (i == 2) {
+                    features.add(new Feature("Feature " + i, Type.CATEGORICAL, new Value("B")));
+                } else {
+                    features.add(new Feature("Feature " + i, Type.NUMBER, new Value(0)));
+                }
+            }
+            background.add(new PredictionInput(features));
+        }
+        ShapConfig shapConfig = testConfig.copy()
+                .withBackground(background)
+                .build();
+        ShapKernelExplainer ske = new ShapKernelExplainer(shapConfig);
+
+        List<Feature> features = new ArrayList<>();
+        for (int i = 0; i < 5; i++) {
+            if (i == 2) {
+                features.add(new Feature(
+                        "Feature " + i,
+                        Type.CATEGORICAL,
+                        new Value("A"),
+                        false,
+                        CategoricalFeatureDomain.create(List.of("A", "B"))));
+            } else {
+                features.add(new Feature("Feature " + i, Type.NUMBER, new Value(i)));
+            }
+        }
+        PredictionInput input = new PredictionInput(features);
+        PredictionProvider model = TestUtils.getTwoOutputSemiCategoricalModel(2);
+        PredictionOutput output = model.predictAsync(List.of(input))
+                .get(Config.INSTANCE.getAsyncTimeout(), Config.INSTANCE.getAsyncTimeUnit())
+                .get(0);
+        Prediction prediction = new SimplePrediction(input, output);
+        ShapResults results = ske.explainAsync(prediction, model)
+                .get(Config.INSTANCE.getAsyncTimeout(), Config.INSTANCE.getAsyncTimeUnit());
+        String table = results.toString();
+        //System.out.println(table);
+        assertTrue(true);
```

Comment:
I would at least assert that the string contains one of the expected keys.
In the worst case scenario I would assert that `table` is not `null`